Tests/SphericalExtractionTest/SphericalExtractionTest.cpp

```diff
@@ -0,0 +1,195 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifdef CH_LANG_CC
+/*
+ *      _______              __
+ *     / ___/ /  ___  __ _  / /  ___
+ *    / /__/ _ \/ _ \/  V \/ _ \/ _ \
+ *    \___/_//_/\___/_/_/_/_.__/\___/
+ *    Please refer to LICENSE, in Chombo's root directory.
+ */
+#endif
+
+// General includes:
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <iomanip>
+#include <iostream>
+#include <sys/time.h>
+
+#include "parstream.H" //Gives us pout()
+using std::endl;
+#include "DefaultLevelFactory.hpp"
+#include "GRAMR.hpp"
+
+#include "GRParmParse.hpp"
+#include "SetupFunctions.hpp"
+#include "SimulationParameters.hpp"
+
+// Problem specific includes:
+#include "AMRInterpolator.hpp"
+#include "Lagrange.hpp"
+#include "SphericalExtraction.hpp"
+#include "SphericalExtractionTestLevel.hpp"
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
```

Comment:
I guess you don't need this here

Reply:
It's possible OpenMP is used in some of the AMR stuff. Although I didn't have problems removing it on my laptop, I will keep it here in case other systems/compilers have issues.